packages/kbn-storybook/lib/theme_switcher.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { Icons, IconButton, TooltipLinkList, WithTooltip } from '@storybook/components';
+import { useGlobals } from '@storybook/api';
+import { Link } from '@storybook/components/dist/tooltip/TooltipLinkList';
+
+const defaultTheme = 'v8.light';
+
+export function ThemeSwitcher() {
+  const [globals, updateGlobals] = useGlobals();
+  const selectedTheme = globals.euiTheme;
+
+  if (!selectedTheme || selectedTheme === defaultTheme) {
+    updateGlobals({ euiTheme: defaultTheme });
+  }
+
+  function Menu({ onHide }: { onHide: () => void }) {
+    const links: Link[] = [
+      {
+        id: 'v8.light',
+        title: 'Amsterdam: Light',
+      },
+      {
+        id: 'v8.dark',
+        title: 'Amsterdam: Dark',
+      },
+      { id: 'v7.light', title: 'Light' },
+      { id: 'v7.dark', title: 'Dark' },
+    ].map((link) => ({
+      ...link,
+      onClick: (_event, item) => {
+        if (item.id !== selectedTheme) {
+          updateGlobals({ euiTheme: item.id });
+        }
+        onHide();
+      },
+      active: selectedTheme === link.id,
+    }));
+
+    return <TooltipLinkList links={links} />;
+  }
+
+  return (
+    <WithTooltip
+      placement="top"
+      trigger="click"
+      closeOnClick
+      tooltip={({ onHide }) => <Menu onHide={onHide} />}
+    >
+      <IconButton key="eui-theme" title="Change the EUI theme">
+        <Icons icon={selectedTheme?.includes('dark') ? 'heart' : 'hearthollow'} />
```

Comment:
The heart is cute, but is there maybe an icon that speaks "theme" more? I clicked all the other buttons before trying this one 😅

Reply:
I couldn't find a place where you can view them all, but this is the list of what's available: https://github.com/storybookjs/storybook/blob/aa05079bef3a088cb1bae51f29fb185b29d100ba/lib/components/src/icon/icons.tsx

Plus the heart has light and dark states and you can imagine it's an
![image](https://user-images.githubusercontent.com/9912/117346552-84f13280-ae6d-11eb-90b4-422fd23a0b72.png).